Dockerfile

```diff
@@ -1,18 +1,10 @@
-FROM maven:3.6.3-jdk-11-openj9 AS MAVEN_BUILD_ENVIRONMENT
-
-COPY pom.xml /tmp/
-COPY src /tmp/src/
-WORKDIR /tmp/
-
-RUN mvn clean package --no-transfer-progress -DskipTests
-
 FROM openjdk:11-jre
```

Comment:
Dockerfile bør helst være en "selv-contained"  dvs. at den har alt for å bygge og kjøre opp en applikasjon. Per nå så har man en avhengighet til at dataservice-catalog.jar er bygget.

Reply:
Jea, men as is blir det strengt tatt aldri brukt. Føler ikke helt gevinsten er stor nok i forhold til å følge best-practice på akkurat dette. Men kan jo la det være opp til @stigbd, jeg skal ikke krangle på å sette dette opp for backend-tjenestene om det er ønskelig.

Reply:
Enig i at docker-fil må vere self-contained. Men eg tenker at det kan vi ta i eit anna issue.

Reply:
her fjerner jeg at docker-fila er self-contained, så tror ikke det trenger å være et annet issue. Men du bør sette opp en epic for ca samtlige backend-tjenester.

Reply:
Fjernet commiten hvor jeg endret på Dockerfile